apps/api/src/routes/index.ts

```diff
@@ -1,14 +1,38 @@
 import { version } from '@/config';
-import { handle } from '@/utils/handle';
+import { isPrismaConnected } from '@/modules/prisma';
+import { handler } from '@/utils/handle';
 import { makeRouter } from '@/utils/routes';
 
+interface Check {
+  name: string;
+  success: boolean;
+}
+
+async function healtcheck(): Promise<Check[]> {
```

Comment:
![image](https://github.com/mrjvs/locali/assets/15330699/4b427a4d-e831-49dc-9553-b3b6f6198a81)


```suggestion
async function healthcheck(): Promise<Check[]> {
```

Reply:
###### Why are you reviewing this pr?

Reply:
Quality control

Reply:
Fixed (after a week, we are enterprise)